infrastructure/sandbox/PreProvisioner/main.tf

```diff
@@ -255,7 +255,7 @@ resource "aws_ecs_task_definition" "main" {
           },
           {
             name  = "QUEUED_INSTANCES"
-            value = "20"
+            value = "5"
```

Comment:
Was this intentional?

Reply:
Ah, I need to make it a different value depending on if its dev/prod. Thank you.